frontend/apps/reader/modules/readertypography.lua

```diff
@@ -66,6 +66,8 @@ local LANGUAGES = {
     { "it",               {"ita"},   "H   ",   _("Italian"),                "Italian.pattern" },
     { "ja",                    {},   "    ",   _("Japanese") },
     { "ko",                    {},   "    ",   _("Korean") },
+    { "la-lit",       {"lat-lit"},   "H   ",   _("Latin (liturgical)"),     "Latin.pattern" },
```

Comment:
Looks like I'm too late, but that should of course be `Latin_liturgical.pattern`. ;-)

Reply:
Going to fix that (no real issue, just a display one, we'll show the hyph filename instead of this pretty language name).
While I'm at it, should these be re-ordered, so the shorter/plain name is first, before the next longer/specific ones?
```diff
     { "ko",                    {},   "    ",   _("Korean") },
-    { "la-lit",       {"lat-lit"},   "H   ",   _("Latin (liturgical)"),
     { "la",               {"lat"},   "H   ",   _("Latin"),
+    { "la-lit",       {"lat-lit"},   "H   ",   _("Latin (liturgical)"),
     { "lv",               {"lav"},   "H   ",   _("Latvian"),
@@ -77,9 +77,9 @@ local LANGUAGES = {
     { "pms",                   {},   "H   ",   _("Piedmontese"),
-    { "pt-BR",                 {},   "HB  ",   _("Portuguese (BR)"),
     { "pt",               {"por"},   "HB  ",   _("Portuguese"),
+    { "pt-BR",                 {},   "HB  ",   _("Portuguese (BR)"),
     { "rm",               {"roh"},   "H   ",   _("Romansh"),
     { "ro",               {"ron"},   "H   ",   _("Romanian"),
+    { "ru",               {"rus"},   "Hb  ",   _("Russian"),
     { "ru-GB",                 {},   "Hb  ",   _("Russian + English (UK)"),
     { "ru-US",                 {},   "Hb  ",   _("Russian + English (US)"),
-    { "ru",               {"rus"},   "Hb  ",   _("Russian"),
     { "sr",               {"srp"},   "HB  ",   _("Serbian"),
```

Reply:
(It needs to be in the current order in crengine to match longer ones first - but in frontend, this order does not matter.)

Reply:
Definitely makes more sense to me.

Reply:
Done with #6913.